7.analyze-coverage/coverage-analysis.py

```diff
@@ -3,7 +3,11 @@
 Interpret Compression
 7.analyze-coverage/coverage-analysis.py
 
-Track gene set coverage across compressed features
+Track gene set coverage across compressed features. We consider a geneset as "covered"
+by the module if it exists as the top feature in the BioBombe applications to the
```

Comment:
Do you mean "if the geneset exists as the top. feature..."?  Not sure how you calculate this?

Reply:
Yeah, geneset. I updated the comment. I think it is more clear now, thanks!